File: app/src/main/java/com/example/eduempoweryd/login/studentregister/StudentRegisterVM.java
package com.example.eduempoweryd.login.studentregister;

import android.os.Bundle;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class StudentRegisterVM extends ViewModel {
    public MutableLiveData<StudentRegisterModel> StudentRegisterModel = new MutableLiveData<>(new StudentRegisterModel());
    private MutableLiveData<Bundle> navArguments = new MutableLiveData<>();

    public MutableLiveData<StudentRegisterModel> getPageTwoModel() {
        return StudentRegisterModel;
    }

    public MutableLiveData<Bundle> getNavArguments() {
        return navArguments;
    }

    public void setNavArguments(Bundle navArguments) {
        this.navArguments.setValue(navArguments);
    }
}

File: jia_hui_s_application/app/src/main/java/com/jiahuisapplication/app/modules/studentregister/data/viewmodel/StudentRegisterVM.kt
package com.jiahuisapplication.app.modules.studentregister.`data`.viewmodel

import android.os.Bundle
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.jiahuisapplication.app.modules.studentregister.`data`.model.SpinnerDescriptionModel
import com.jiahuisapplication.app.modules.studentregister.`data`.model.StudentRegisterModel
import kotlin.collections.MutableList
import org.koin.core.KoinComponent

class StudentRegisterVM : ViewModel(), KoinComponent {
  val studentRegisterModel: MutableLiveData<StudentRegisterModel> =
      MutableLiveData(StudentRegisterModel())

  var navArguments: Bundle? = null

  val spinnerDescriptionList: MutableLiveData<MutableList<SpinnerDescriptionModel>> =
      MutableLiveData()
}
